CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing
+
+Thanks for being willing to contribute!
+
+Working on your first Pull Request? You can learn how from this free series
+[How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+## Project setup
+
+1. Fork this repository
+2. Clone your forked repository
+3. Run `npm install` to install corresponding dependencies
+4. Create a branch for your PR named like `pr/your-branch-name` (you can do this through git CLI with `git checkout -b pr/your-branch-name`)
```

Comment:
It would be useful to add a note on the convention we have for the pull requests: adding the type (`feat`, `fix`, ...), etc.

Reply:
There is nothing at the moment forcing anyone to create the branch following a namespace, so I kept it simple here.